openrr-nav/examples/dwa.rs

```diff
@@ -68,6 +68,8 @@ fn main() {
     maps.insert(GOAL_DISTANCE_MAP_NAME.to_owned(), goal_distance_map);
     maps.insert(OBSTACLE_DISTANCE_MAP_NAME.to_owned(), obstacle_distance_map);
     let layered = LayeredGridMap::new(maps);
+    let angles = HashMap::new();
+    // TODO: Add angles
```

Comment:
I can't understand what TODO is from this line.
If you need some time to implement the TODO, please add more description to explain the task more clearly.

Reply:
A memo was left behind.